src/main/java/com/targomo/client/api/request/config/RequestHeaderFilter.java

```diff
@@ -0,0 +1,24 @@
+package com.targomo.client.api.request.config;
+
+import javax.ws.rs.client.ClientRequestContext;
+import javax.ws.rs.client.ClientRequestFilter;
+import java.io.IOException;
+
+/**
+ * @author Mario Freitas
+ */
+public abstract class RequestHeaderFilter implements ClientRequestFilter {
```

Comment:
I would add a small comment about what this is for/how it is used.

Reply:
Done